packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -54,25 +54,7 @@ export class OrgList implements vscode.Disposable {
     const authFilesArray = await AuthInfo.listAllAuthorizations().catch(
       err => null
     );
-
-    if (authFilesArray === null || authFilesArray.length === 0) {
-      return null;
-    }
-    const authInfoObjects: FileInfo[] = [];
-    for (const authFile of authFilesArray) {
-      try {
-        const filePath = path.join(
-          await ConfigFile.resolveRootFolder(true),
-          '.sf',
-          authFile.username
-        );
-        const fileData = readFileSync(filePath, 'utf8');
-        authInfoObjects.push(JSON.parse(fileData));
-      } catch (e) {
-        console.log(e);
-      }
-    }
-    return authInfoObjects;
+    return authFilesArray;
   }
 
   public async filterAuthInfo(authInfoObjects: FileInfo[]) {
```

Comment:
Note: there is a filter option on `listAllAuthorizations` we cold utilize if we wanted.

Reply:
just documenting that on line 80 below from the old PR `That field indicates that the file represents a new user, other than the default admin, within the org.` We should probably verify this is still valid